exporter/SynthesisFusionAddin/src/Parser/ExporterOptions.py

```diff
@@ -51,7 +51,7 @@ class ExporterOptions:
     autoCalcGamepieceWeight: bool = field(default=False)
 
     frictionOverride: bool = field(default=False)
-    frictionOverrideCoeff: float | None = field(default=None)
+    frictionOverrideCoeff: float = field(default=0.5)
```

Comment:
This was causing the slider to always show up?

Reply:
Would cause the `__init__` of `GeneralOptionsTab` to crash before it could set the visibility of the slider since the Fusion API was not expecting to see a value of `None` for the slider position.